R/statistic-histogram.R

```diff
@@ -2,33 +2,52 @@
 #' 
 #' Determine accuracy of histogram release, given epsilon and delta, for the differentially 
 #' private histogram release.
+#' 
+#' In differential privacy, "accuracy" is defined as the threshold value above which a given value 
+#' is "significantly different" from the expected value. Mathematically, this is written as:
+#' \deqn{\alpha = Pr[Y > a]}
+#' Where \eqn{\alpha} is the statistical significance level, \eqn{a} is the accuracy,
+#' and \eqn{Y} is a random  variable indicating the difference between the differentially 
```

Comment:
Better to include equation for Y I think

Reply:
what does this mean?